### ISSUE TITLE
Adding Telegram as a report method

### ISSUE THREAD

**Issue by juanvelascogomez** [captainGeech42/ransomwatch#42]
Hello! I would like to congratulate you on this project. Have you think about adding a Telegram as a report method?

Let me know what you think about this!

Best,

**Comment by captainGeech42** [comment]
Hmm, interesting. I think there could be value there, but I haven't done any work with Telegram so I don't know how feasible it is.

Do you have experience doing chatbot-type things with Telegram?

**Comment by juanvelascogomez** [comment]
Yes, i've worked with Telethon project. It seems the best way to achieve this via bot, for example.
